OpenAdaptAI/OpenAdapt
# Web dashboard

**What kind of change does this PR introduce?**


This PR adds a web dashboard to the tray. The tray now contains a 'Launch dashboard' option, clicking on which will start a nextjs server at port 3000, and a fastapi server at 8000. A new browser tab will be opened with the nextjs app url. On exiting the tray, the servers are gracefully shutdown.

To complete the initial setup, see the changes to the `Manual Setup` section in `README.md`

**Summary**



**Checklist**

**How can your code be run and tested?**


To complete the initial setup, see the changes to the `Manual Setup` section in `README.md`

<details><summary>Screenshot</summary>
<img width="2048" alt="image" src="https://github.com/OpenAdaptAI/OpenAdapt/assets/40872556/f314b4d4-afb1-425a-9b8a-4d4147019a23">
</details>
<details><summary>Screen recoding</summary>
[Google drive link](https://drive.google.com/file/d/18L3tR2IHKlRvU2XICfMphomd8-dQm-am/view?usp=drive_link)
</details>
